### ISSUE TITLE
File report bug

### ISSUE THREAD

**Issue by helio-frota** [bucharest-gold/szero#20]
```
[ Declaration and file lines ]
undefined:
./from-stream.js:5
./from-stream.js:5
./from-stream.js:41
./from-stream.js:4
./from-stream.js:4
./from-stream.js:29
./index.js:6
./index.js:6
./index.js:25
./index.js:49
./index.js:51
./index.js:55
./index.js:49
./index.js:7
./index.js:9
./index.js:10
./index.js:25
./index.js:44
./index.js:49
./index.js:51
./index.js:53
./index.js:60
./index.js:61
./lib/contextify.js:3
./lib/contextify.js:3
./lib/contextify.js:147
./lib/contextify.js:147
./lib/contextify.js:151
./lib/contextify.js:152
./lib/svg-template.js:3
./lib/svg-template.js:3
./lib/svg-template.js:6
./lib/svg-template.js:8
./lib/svg.js:3
./lib/svg.js:3
./lib/svg.js:31
./lib/svg.js:31
./lib/svg.js:6
./lib/svg.js:8
./lib/svg.js:17
./lib/svg.js:27
./lib/svg.js:30
./lib/svg.js:31
./lib/svg.js:32
./lib/svg.js:33

[ Declaration and amount ]
undefined --> [45]

[ Unused dependencies ]
```

**Comment by helio-frota** [comment]
Also difference between console and file report:

```
[hf@archT440 roi]$ szero .
----------------------------------------------------------------------
[ Declaration and file lines ]
----------------------------------------------------------------------
require('fidelity'):
./index.js:32
Fidelity:
./index.js:32
./index.js:113
./index.js:145
./index.js:178
./index.js:208
./index.js:236
./index.js:264
./index.js:291
----------------------------------------------------------------------
[ Declaration and amount ]
----------------------------------------------------------------------
require('fidelity') --> [ 1 ]
Fidelity --> [ 8 ]
----------------------------------------------------------------------
[ Unused dependencies ]
----------------------------------------------------------------------
fidelity,fidelity,fidelity,fidelity,fidelity,fidelity,fidelity,fidelity


[hf@archT440 roi]$ cat szero.txt 
[ Declaration and file lines ]
undefined:
./index.js:32
./index.js:32
./index.js:113
./index.js:145
./index.js:178
./index.js:208
./index.js:236
./index.js:264
./index.js:291

[ Declaration and amount ]
undefined --> [9]

[ Unused dependencies ]
```

**Comment by helio-frota** [comment]
Works on tests:

```
    [ Declaration and file lines ]
    require('roi'):
    x.js:4
    roi:
    x.js:4
    x.js:7
    [ Declaration and amount ]
    require('roi') --> [1]
    roi --> [2]
    [ Unused dependencies ]
    fidelity
```

Not working when using command line:

```
[ Declaration and file lines ]
undefined:
./foo/x.js:4
./foo/x.js:4
./foo/x.js:7
./xpto/abc/w.js:4
./xpto/abc/w.js:4

[ Declaration and amount ]
undefined --> [5]

[ Unused dependencies ]
```

**Comment by lholmquist** [comment]
this should fix this 778c8df4714db415242438fabc308efe492abf96
